tests/ballerina-compiler-api-test/src/test/java/io/ballerina/semantic/api/test/ExpressionTypeTest.java

```diff
@@ -325,6 +326,26 @@ public void testFunctionCall(int sLine, int sCol, int eLine, int eCol, TypeDescK
         };
     }
 
+    @Test
+    public void testExpressionsOfIntersectionTypes() {
+        assertType(135, 4, 135, 21, INTERSECTION);
+        assertType(135, 4, 135, 22, INTERSECTION);
```

Comment:
I don't think so, since what we're testing is the expression's type. I guess even two here is not absolutely necessary, but followed the pattern as with the other tests.

I assume there'd be separate tests that test identifying the expression for different combinations of line/column.